src/modules/user/controller/userControllers.ts

```diff
@@ -0,0 +1,37 @@
+// user Controllers
+import { Request, Response } from "express";
+
+import authRepositories from "../../auth/repository/authRepositories";
+import httpStatus from "http-status";
+
+const updateUserRole = async (req: Request, res: Response) => {
+  try {
+    await authRepositories.UpdateUserByAttributes("role", req.body.role, "id", req.params.id)
+    const updatedUser = await authRepositories.findUserByAttributes("id", req.params.id)
```

Comment:
In UpdateUserByAttributes we already returned findUserByAttribute. Why are you calling it back here.